src/library_openal.js

```diff
@@ -468,29 +468,29 @@ var LibraryOpenAL = {
     updateListenerSpace: function(ctx) {
       var listener = ctx.audioCtx.listener;
       if (listener.positionX) {
-        listener.positionX.value = listener._position[0];
-        listener.positionY.value = listener._position[1];
-        listener.positionZ.value = listener._position[2];
+        listener.positionX.value = ctx.listener.position[0];
+        listener.positionY.value = ctx.listener.position[1];
+        listener.positionZ.value = ctx.listener.position[2];
       } else {
 #if OPENAL_DEBUG
         warnOnce('Listener position attributes are not present, falling back to setPosition()');
 #endif
-        listener.setPosition(listener._position[0], listener._position[1], listener._position[2]);
+        listener.setPosition(ctx.listener.position[0], ctx.listener.position[1], ctx.listener.position[2]);
       }
       if (listener.forwardX) {
-        listener.forwardX.value = listener._direction[0];
-        listener.forwardY.value = listener._direction[1];
-        listener.forwardZ.value = listener._direction[2];
-        listener.upX.value = listener._up[0];
-        listener.upY.value = listener._up[1];
-        listener.upZ.value = listener._up[2];
+        listener.forwardX.value = ctx.listener.direction[0];
+        listener.forwardY.value = ctx.listener.direction[1];
+        listener.forwardZ.value = ctx.listener.direction[2];
+        listener.upX.value = ctx.listener.up[0];
+        listener.upY.value = ctx.listener.up[1];
+        listener.upZ.value = ctx.listener.up[2];
       } else {
 #if OPENAL_DEBUG
         warnOnce('Listener orientation attributes are not present, falling back to setOrientation()');
 #endif
         listener.setOrientation(
-          listener._direction[0], listener._direction[1], listener._direction[2],
-          listener._up[0], listener._up[1], listener._up[2]);
+        		ctx.listener.direction[0], ctx.listener.direction[1], ctx.listener.direction[2],
```

Comment:
indentation looks wrong here

Reply:
Indeed, looks like my editor wasn't setup properly for 2 spaces indentations